### ISSUE TITLE
error handling in dll

### ISSUE THREAD

**Issue by CarolineMathieson** [project-everest/mitls-fstar#213]
When calling any of the FFI_mitls_configure_xxx functions to configure cipher suites, signature algorithms or named groups, the functions are very sensitive to the precise names used to specify the values. Since these are passed by name, the exact name must be used. When there is a slight error, the dll typically crashes the application using the dll without providing any error message.  So your application just disappears without warning. 

Error handling needs to be improved so that in this situation, an error indication is given and the function should just return 0 as the other API functions do.

**Comment by ad-l** [comment]
Agreed it's a problem. The configuration functions currently can't return errors
